package.json

```diff
@@ -37,67 +41,68 @@
   },
   "license": "MIT",
   "funding": "https://github.com/sponsors/philsturgeon",
-  "main": "lib/index.js",
-  "typings": "lib/index.d.ts",
-  "exports": {
```

Comment:
Will this provide the same ES modules + CJS support that was added in #295? I don't want to annoy older users, but perhaps if we've raised the node version high enough then thats moot now anyway?

Reply:
Yes, I moved this back to being a cjs package. I also changed the pinned version from node experimental (17) to LTS (16). 14 will EOL in April so we should still support 16.